--- a/_internal/constants.py
+++ b/_internal/constants.py
@@ -24,7 +24,2 @@
 CLI_DEFAULTS: Dict[str, Any] = dict(  # pylint: disable=use-dict-literal
-    config_files=[
-        os.path.join(misc.get_default_folder('config'), 'cli.ini'),
-        # https://freedesktop.org/wiki/Software/xdg-user-dirs/
-        os.path.join(os.environ.get("XDG_CONFIG_HOME", "~/.config"),
-                     "letsencrypt", "cli.ini"),
-    ],
+    config_files=[],
