--- a/_internal/constants.py	2024-01-01 00:00:00.000000000 +0000
+++ b/_internal/constants.py	2024-01-01 00:00:00.000000000 +0000
@@ -15,12 +15,7 @@
 """Plugins Setuptools entry point before rename."""
 
 CLI_DEFAULTS = dict(
-    config_files=[
-        os.path.join(misc.get_default_folder('config'), 'cli.ini'),
-        # http://freedesktop.org/wiki/Software/xdg-user-dirs/
-        os.path.join(os.environ.get("XDG_CONFIG_HOME", "~/.config"),
-                     "letsencrypt", "cli.ini"),
-    ],
+    config_files=[],
 
     # Main parser
     verbose_count=-int(logging.INFO / 10),
