--- a/certbot/_internal/constants.py
+++ b/certbot/_internal/constants.py
@@ -22,12 +22,7 @@
 """Plugins Setuptools entry point before rename."""
 
 CLI_DEFAULTS: Dict[str, Any] = dict(  # pylint: disable=use-dict-literal
-    config_files=[
-        os.path.join(misc.get_default_folder('config'), 'cli.ini'),
-        # https://freedesktop.org/wiki/Software/xdg-user-dirs/
-        os.path.join(os.environ.get("XDG_CONFIG_HOME", "~/.config"),
-                     "letsencrypt", "cli.ini"),
-    ],
+    config_files=[],
 
     # Main parser
     verbose_count=0,
